src/HomeScreen.js

```diff
@@ -7,45 +7,32 @@ import PlayerStats from './PlayerStats.js'
 export const socket = io(); // Connects to socket connection
 
 export default function Home ({playerName}) {
-  //state for joining multiplayer room or not
-  const [playerJoinedMultiplayer, setPlayerJoinedMultiplayer] = useState(false)
-  //state list of all players in all the rooms
-  const [activePlayers, setActivePlayers] = useState([])
-  const room = 'Multiplayer'
+  const [playerStartedGame, setPlayerStartedGame] = useState(false) // State for joining multiplayer room or not
+  const [activePlayers, setActivePlayers] = useState([]) // State list of all players in all the rooms
+  const [room, setRoom] = useState(""); // State for keeping track of the room the player is in
 
-  const leaveRoom = ()=>{
-    //when someone clicks the leave room button
-    socket.emit('leaveRoom', {playerName, room})
-    setPlayerJoinedMultiplayer(false)
-  }
-
-  const joinRoom = ()=>{
```

Comment:
Removing this for now because all players will auto-join the "Multiplayer" lobby for sprint 1.